src/profile-logic/marker-data.js

```diff
@@ -116,15 +119,25 @@ export function getSearchFilteredMarkerIndexes(
   if (!searchRegExp) {
     return markerIndexes;
   }
+  const categoryList = getDefaultCategories();
   const newMarkers: MarkerIndex[] = [];
   for (const markerIndex of markerIndexes) {
-    const { data, name } = getMarker(markerIndex);
+    const { data, name, category } = getMarker(markerIndex);
 
     // Reset regexp for each iteration. Otherwise state from previous
     // iterations can cause matches to fail if the search is global or
     // sticky.
+
     searchRegExp.lastIndex = 0;
 
+    if (categoryList[category - 1] !== undefined) {
```

Comment:
I believe you used `category - 1` because the list of categories was incorrect, as I commented above. Once you'll get the right list of categories, you'll be able to use `[category]`, which is the correct thing to do.

Reply:
changes made in recent push